aries_cloudagent/connections/models/diddoc_v2/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""
+DID Document model support.
+
+Copyright 2021 Sicpa
```

Comment:
not sure about this SICPA copyright , @dbluhm ?

Reply:
I'm no copyright lawyer but yeah, I think our DCO sign off is basically saying that we forfeit the ability to claim copyright on the code after it's been committed to the project

Reply:
Which actually reminds me; @Luis-GA, before we can merge your changes, each of your commits need to include the `Signed-off-by: Your Name <your@email.com>` separated by an empty line at the end of your commit messages. When you commit in the future, include the `-s` flag which will automatically add this line to the commit.

To add the sign off to your previous commits, you will have to do an interactive rebase, editting the commit message to include the sign off.

Reply:
So do you suggest to remove it? 

I made this due this https://github.com/rd-dlab/aries-cloudagent-python/blob/master/aries_cloudagent/connections/models/diddoc/diddoc.py#L4